src/customer/strategies/masterpass-customer-strategy.spec.ts

```diff
@@ -0,0 +1,188 @@
+import { createRequestSender } from '@bigcommerce/request-sender';
+import { createScriptLoader } from '@bigcommerce/script-loader';
+
+import { getCartState } from '../../cart/carts.mock';
+import { createCheckoutStore, CheckoutStore } from '../../checkout';
+import { getCheckoutState } from '../../checkout/checkouts.mock';
+import { NotInitializedError } from '../../common/error/errors';
+import InvalidArgumentError from '../../common/error/errors/invalid-argument-error';
```

Comment:
Prefer importing from the index file when available.

In this case I think this same getter is available in: ../../common/errors.

NOTE: This applies to the whole PR.